Misc/NEWS.d/next/Core_and_Builtins/2025-01-22-14-24-44.gh-issue-129149.wAYu43.rst

```diff
@@ -0,0 +1,2 @@
+Add missing fast path in :c:func:`PyLong_FromUnsignedLong`, :c:func:`PyLong_FromUnsignedLongLong` and :c:func:`PyLong_FromSize_t` functions
+for compact integers.
```

Comment:
```suggestion
Add missing fast path for medium-size integers in :c:func:`PyLong_FromLong`.
```

Reply:
`PyLong_FromLong` already contains fast path for medium-size integers https://github.com/python/cpython/blob/main/Objects/longobject.c#L329-L330

Reply:
I have addressed `for medium-size integers` [5760c95](https://github.com/python/cpython/pull/129168/commits/5760c954fd39b9787a6616db4efe967a67d6ef39)